.github/workflows/ci.yml

```diff
@@ -299,6 +299,91 @@ jobs:
           gcloud storage cp latest.txt $BUCKET_URI/latest.txt
           gcloud storage cp $BUCKET_URI/bitstream-${{ github.sha }}.tar.gz $BUCKET_URI/bitstream-latest.tar.gz
 
+  # CW310 FPGA jobs.
+  execute_test_rom_fpga_tests_cw310:
+    name: CW310 Test ROM Tests
+    needs: [chip_earlgrey_cw310, sw_build_test]
```

Comment:
I think we discussed dropping the `sw_build_test` dependency, did we agree to keep it?